src/modules/likesApi.js

```diff
@@ -0,0 +1,46 @@
+export const postLike = async (likeData) => {
+  try {
+    const response = await fetch(
+      'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Et4qFKlhg2AwekIfP1aj/likes',
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/Json',
+          charset: 'utf-8',
+        },
+        body: JSON.stringify(likeData),
+      },
+    );
+
+    if (!response.ok) {
+      return false;
+    }
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    return false;
+  }
+};
+
+export const getLikes = async () => {
+  const getStatus = await fetch(
+    'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Et4qFKlhg2AwekIfP1aj/likes',
+    {
+      method: 'GET',
+    },
+  );
+  const commentResponse = await getStatus.json();
+  const likesArray = Array.isArray(commentResponse) ? commentResponse : [];
+  return likesArray;
+};
+
+// export const getLikes = async () => {
+//   const getStatus = await fetch(
+//     'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Et4qFKlhg2AwekIfP1aj/likes',
+//     {
+//       method: 'GET',
+//     }
+//   );
+//   const commentResponse = await getStatus.json();
+//   return commentResponse;
+// };
```

Comment:
- In my opinion, it is better to have all the commted-out lines of code removed to make your code clean :recycle: